packages/fast-components-class-name-contracts-msft/src/button/index.ts

```diff
@@ -0,0 +1,12 @@
+import { IButtonClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+/**
+ * The class name contract for the button component
+ */
+export interface IMSFTButtonClassNameContract extends IButtonClassNameContract {
```

Comment:
@nalogiudice Are we able to rename this import as well for alignment?

Reply:
I actually think this is more correct. It *IS* the MSFT contract. I can change obviously. @janechu any thoughts?